src/main/java/ru/yandex/clickhouse/ClickHouseDatabaseMetadata.java

```diff
@@ -856,7 +856,9 @@ public ResultSet getColumns(String catalog, String schemaPattern, String tableNa
                 ? String.valueOf(columnNullable)
                 : String.valueOf(columnNoNulls));
             //remarks
-            row.add(null);
+            row.add("".equals(descTable.getString("comment"))
```

Comment:
Why we treat empty string as null? If that's required, can use a local variable to store comment so that we don't have to call getString twice?

Reply:
ClickHouse returns an empty String for columns without a comment (column is not `Nullable`). Why I think that translating this empty String to `null` is a good behavior in this case:

1. A typical Java user would expect this to be `null` when no comment has been set.
2. Returning an empty String for a column which does not have a comment set would be a change in this driver's behavior. This is covered by an existing test, too.

I removed the duplicate call to `getString`.